common/network_information.go

```diff
@@ -0,0 +1,27 @@
+// pmm-agent
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Package common provides common types used in many packages.
+package common
+
+import "time"
+
+// NetworkInformation contains data about ping, time drift and roundtrip.
+type NetworkInformation struct {
+	ClockDrift time.Duration
+	Ping       time.Duration
+	Roundtrip  time.Duration
+}
```

Comment:
Let's return two durations from `getNetworkInformation` (clock drift and latency) and remove that type and "common" package.
In general: https://dave.cheney.net/practical-go/presentations/qcon-china.html#_avoid_package_names_like_base_common_or_util Sometimes it is hardly avoidable, but in that case, we can.

Reply:
Why did we decide to not have dependency on client on agent_local?

Reply:
IIRC, there was a circular dependency. Then mock enabled easier testing.

Reply:
> I've made a PR to this, maybe it can help?? 

Sorry, I don't think two values we can return from function directly are worth it.